jib-maven/jib-maven.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: jib-maven
+spec:
+  inputs:
+    params:
+    - name: IMAGE
+      description: The name of the image to push
+    - name: DIRECTORY
+      description: The directory containing the app, relative to the source repository root
+      default: .
+    - name: CACHE
+      description: The name of the volume for caching Maven artifacts and base image layers
+      default: empty-dir-volume
+
+    resources:
+    - name: source
+      type: git
+
+  steps:
+  - name: build-and-push
+    image: gcr.io/cloud-builders/mvn
+    command:
+    - mvn
+    - compile
+    - com.google.cloud.tools:jib-maven-plugin:build
+    - -Duser.home=/builder/home
+    - -Dimage=${inputs.params.IMAGE}
+    workingDir: /workspace/source/${inputs.params.DIRECTORY}
+    volumeMounts:
+    - name: ${inputs.params.CACHE}
+      mountPath: /builder/home/.m2
+      subPath: m2-cache
+    - name: ${inputs.params.CACHE}
+      mountPath: /builder/home/.cache
+      subPath: jib-cache
+
+  volumes:
+  - name: empty-dir-volume
+    emptyDir: {}
+
+---
+
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
```

Comment:
Can you move the task run to a new file?

Reply:
I think we should just remove this task run section, that's what I did for the other Tasks. The example task is in the README, where users hopefully won't be confused when they apply all YAMLs in the directory.